cluster/aws/templates/salt-minion.sh

```diff
@@ -61,6 +61,8 @@ if [[ -n "${KUBELET_ROOT}" ]]; then
 EOF
 fi
 
-install-salt
+apt-get autoclean
```

Comment:
Can you add a comment about why this is important?  It isn't obvious to me...  (just disk space?)